concourse/pipelines/destroy-cloudfoundry.yml

```diff
@@ -103,7 +117,17 @@ jobs:
             passed: ['init']
             trigger: true
           - get: bosh-secrets
+          - get: bosh-CA
           - get: paas-cf
+          - get: cf-secrets
+          - get: cf-manifest
+
+      - task: get-cf-cli-config
+        file: paas-cf/concourse/tasks/get-cf-cli-config.yml
+
+      - task: remove-healthcheck-app
+        file: paas-cf/concourse/tasks/remove-healthcheck-app.yml
```

Comment:
As we are deleting CF, the application will get deleted with all the platform, so there is no need to delete it during the destroy.